docs/benchmarks.rst

```diff
@@ -42,26 +42,18 @@ Example output and comparison of different runmodes
 
 .. code-block::
 
-     -- Running Simple benchmark with StrictRedis lib and non cluster server --
-    python benchmarks/simple.py --port 7007 --timeit --nocluster
-    10.0k SET/GET operations took: 0.9711470603942871 seconds... 10297.10165208139 operations per second
-    20.0k SET/GET operations took: 1.9136295318603516 seconds... 10451.343725113202 operations per second
-    40.0k SET/GET operations took: 3.8409764766693115 seconds... 10414.018477584079 operations per second
-
-     -- Running Simple benchmark with StrictRedisCluster lib and cluster server --
-    python benchmarks/simple.py --port 7001 --timeit
-    10.0k SET/GET operations took: 0.760077714920044 seconds... 13156.549394494412 operations per second
-    20.0k SET/GET operations took: 1.5251967906951904 seconds... 13113.061948474155 operations per second
-    40.0k SET/GET operations took: 3.05112361907959 seconds... 13109.924406165655 operations per second
+     -- Running Simple benchmark with StrictRedis lib and non cluster server, 50 concurrent processes and total 50000*2 requests --
+    python benchmarks/simple.py --host 172.16.166.31 --timeit --nocluster -c 50 -n 50000
```

Comment:
Change from `--host 172...` to `--host 127.0.0.1 --port 7000` Can't have your IP set in the docs. Do this for all occurrences in this file.